command/plugin.go

```diff
@@ -145,7 +146,9 @@ var Builders = map[string]packer.Builder{
 	"parallels-iso":       new(parallelsisobuilder.Builder),
 	"parallels-pvm":       new(parallelspvmbuilder.Builder),
 	"profitbricks":        new(profitbricksbuilder.Builder),
-	"proxmox":             new(proxmoxbuilder.Builder),
+	"proxmox-clone":       new(proxmoxclonebuilder.Builder),
+	"proxmox-iso":         new(proxmoxisobuilder.Builder),
+	"proxmox":             new(proxmoxisobuilder.Builder),
```

Comment:
@SwampDragons I did this as a backwards-compat thing (letting the old builder string remain as an alias). Good or bad? Is there a deprecation policy for this?

Reply:
Yeah, this seems like a good idea until the v1.7 release. We try only to break backwards compat in the minor releases, not the patch ones. Then it can be up to you whether to leave it in or remove it and force the upgrade. You can write a fixer to make the upgrade path trivial.